x-pack/plugins/security_solution/public/app/solution_navigation/links/sections/ml_links.ts

```diff
@@ -179,6 +180,12 @@ export const mlNavLinks: SolutionNavLink[] = [
     landingIcon: IconDataViewLazy,
     description: i18n.INDEX_DATA_VISUALIZER_DESC,
   },
+  {
+    id: ExternalPageName.mlESQLdataVisualizer,
+    title: i18n.ESQL_DATA_VISUALIZER_TITLE,
+    landingIcon: IconDataViewLazy,
```

Comment:
Nit - is there a different icon we could use here to differentiate it from the data view data visualizer icon?

eui have this `sqlApp` icon for example, but I don't know if there is something similar from the icon set used by the security nav?

<img width="229" alt="Screenshot 2024-09-12 at 10 52 42" src="https://github.com/user-attachments/assets/84544fb3-8c2b-4280-a11c-941f67cd9f0b">

Reply:
Reached out regarding this question and will update once I have an update.

Reply:
Hey @JasonStoltz @yanwalton, would you be able to suggest an icon that we could use here? Is it ok if we use the same icon?

Reply:
@alvarezmelissa87 @peteharverson I think it makes sense to go with the sqlApp icon for now. If it needs to be swapped out later, it can be, but I think it make sense to use one of the duotone icons to keep it consistent. 

cc @MichaelMarcialis @ryankeairns

Reply:
The icon has been updated to 'sqlApp' and the screenshots in the description have been updated.